lib/internal/wrap_js_stream.js

```diff
@@ -72,6 +72,8 @@ class JSStreamWrap extends Socket {
     this.stream = stream;
     this[kCurrentWriteRequest] = null;
     this[kCurrentShutdownRequest] = null;
+    this.readable = true;
+    this.writable = true;
```

Comment:
Just a thought, but I think it might be more correct to forward these values from `stream` itself?